WebDriverAgentLib/Routing/FBElementUtils.m

```diff
@@ -12,16 +12,33 @@
 #import "FBElementUtils.h"
 #import "FBElementTypeTransformer.h"
 
+NSString *const FBUnknownAttributeException = @"FBUnknownAttributeException";
 static NSString *const WD_PREFIX = @"wd";
+static NSString *const OBJC_PROP_GETTER_PREFIX = @"G";
+static NSString *const OBJC_PROP_ATTRIBS_SEPARATOR = @",";
 
 @implementation FBElementUtils
 
 + (NSString *)wdAttributeNameForAttributeName:(NSString *)name
 {
-  if ([name hasPrefix:WD_PREFIX]) {
```

Comment:
This is heavy method that is called really often. I would strongly consider building up cache for attribute names, because they will never change, right?